src/test/test_proj_data_info.cxx

```diff
@@ -293,7 +293,7 @@ ProjDataInfoTests::test_generic_proj_data_info(ProjDataInfo& proj_data_info)
                         || !check(diff_view_num <= 1, "round-trip get_LOR then get_bin (LORAs2Points): view")
                         || !check(diff_axial_pos_num <= 1, "round-trip get_LOR then get_bin (LORAs2Points): axial_pos")
                         || !check(diff_tangential_pos_num <= 1, "round-trip get_LOR then get_bin (LORAs2Points): tangential_pos")
-                        || !check(diff_timing_pos_num == 0, "round-trip get_LOR then get_bin: timing_pos"))
+                        || !check(diff_timing_pos_num == 0 || diff_timing_pos_num ==  proj_data_info.get_max_tof_pos_num(), "round-trip get_LOR then get_bin: timing_pos"))
```

Comment:
as above

Reply:
I have to say I do not remember but if I remove it it fails. need to think about why